jdk-javac-plugin/README.md

```
@@ -0,0 +1,37 @@
+`com.uber.nullaway.jdkannotations.AstubxGeneratorCLI`) that turns the `.json`
+files into an `.astubx` file.  
+
+Here are the current steps to (re-)generate the file (admittedly janky, we will
+work to improve them).
+
+1. Build this module and the `astubx-generator-cli` module:
+`./gradlew :jdk-javac-plugin:build :jdk-annotations:astubx-generator-cli:build`
+2. Clone [this fork](https://github.com/msridhar/jdk) of the JSpecify JDK, and
+check out the `test` branch.
+3. In the jdk repo, edit these lines of `make/common/JavaCompilation.gmk`:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)